run.py

```diff
@@ -3,7 +3,6 @@
 import sys
 import logging
 import logging.config
-from slackbot import settings
 from slackbot.bot import Bot
```

Comment:
prob legit to keep this killed if nothing in the original implementation is using it